candi/bashible/bundles/centos-7/bootstrap.sh.tpl

```diff
@@ -38,10 +38,9 @@ REGISTRY_AUTH="$(base64 -d <<< "{{ .registry.auth }}")"
 REGISTRY_AUTH="$(base64 -d <<< "{{ .registry.dockerCfg }}" | python -c 'import json; import sys; dockerCfg = sys.stdin.read(); parsed = json.loads(dockerCfg); parsed["auths"]["'${REGISTRY_ADDRESS}'"].setdefault("auth", ""); print(parsed["auths"]["'${REGISTRY_ADDRESS}'"]["auth"]);' | base64 -d)"
 {{- end }}
 BB_RP_INSTALLED_PACKAGES_STORE="/var/cache/registrypackages"
-BB_RP_PREFIX="${REGISTRY_PATH%/*}/binaries"
 {{- /*
 # check if image installed
-# bb-rp-is-installed? image tag
+# bb-rp-is-installed? package tag
 */}}
 bb-rp-is-installed?() {
```

Comment:
Proposal to discuss:
* write a common library that uses python or something to parse yaml.
* use it in bashible.
* `include` it here with helm.

Reply:
Add comment about migration to centos 8